src/plugins/intel_npu/src/plugin/npuw/weights_bank.cpp

```diff
@@ -0,0 +1,67 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "weights_bank.hpp"
+
+using ov::npuw::weights::Bank;
+
+class BankManager {
+public:
+    static BankManager& getInstance() {
+        static BankManager instance;
+        return instance;
+    }
+
+private:
+    BankManager() {}
+    BankManager(BankManager const&);
+    void operator=(BankManager const&);
```

Comment:
is it `delete`? is it `default`? is it defined elsewhere?

Reply:
Done